lib/ansible/modules/network/nxos/nxos_logging.py

```diff
@@ -458,8 +724,14 @@ def main():
         dest_level=dict(type='int', aliases=['level']),
         facility_level=dict(type='int'),
         interface=dict(),
+        facility_link_status=dict(choices=['link-down-notif', 'link-down-error', 'link-up-notif', 'link-up-error']),
+        event=dict(choices=['link-enable', 'link-default', 'trunk-enable', 'trunk-default']),
+        message=dict(choices=['add-interface-description']),
+        file_size=dict(type='int'),
+        timestamp=dict(choices=['microseconds', 'milliseconds', 'seconds']),
         state=dict(default='present', choices=['present', 'absent']),
-        aggregate=dict(type='list')
+        aggregate=dict(type='list'),
+        purge=dict(default=False, type='bool')
```

Comment:
I remember we discussed adding the `purge` option to the default `nxos_argument_spec`.  Is this something we should do now?

Reply:
This is where I had it initially but decided to make it module specific for now.  I think when we start implementing purge for other nxos modules we can add it to the nxos_argument_spec.  IMO.